clippy_lints/src/inline_fn_without_body.rs

```diff
@@ -37,12 +37,9 @@ impl LintPass for Pass {
 
 impl<'a, 'tcx> LateLintPass<'a, 'tcx> for Pass {
     fn check_trait_item(&mut self, cx: &LateContext<'a, 'tcx>, item: &'tcx TraitItem) {
-        match item.node {
-            TraitItemKind::Method(_, TraitMethod::Required(_)) => {
-                check_attrs(cx, &item.name, &item.attrs);
-            },
-            _ => {},
-        }
+        if let TraitItemKind::Method(_, TraitMethod::Required(_)) = item.node {
+    check_attrs(cx, &item.name, &item.attrs);
```

Comment:
formatting fail

Reply:
Interesting, it was created with rustfix from misformatted clippy suggestion.
I was sure rustfmt would fix formatting, I'm gonna be more careful with it now.

```
warning: you seem to be trying to use match for destructuring a single pattern. Consider using `if let`
  --> src/inline_fn_without_body.rs:40:9
   |
40 | /         match item.node {
41 | |             TraitItemKind::Method(_, TraitMethod::Required(_)) => {
42 | |                 check_attrs(cx, &item.name, &item.attrs);
43 | |             },
44 | |             _ => {},
45 | |         }
   | |_________^
   |
   = note: #[warn(single_match)] on by default
   = help: for further information visit https://rust-lang-nursery.github.io/rust-clippy/v0.0.187/index.html#single_match
help: try this
   |
40 |         if let TraitItemKind::Method(_, TraitMethod::Required(_)) = item.node {
41 |     check_attrs(cx, &item.name, &item.attrs);
42 | }
   |
```